OpenRA.Mods.Common/Widgets/Logic/Editor/MapEditorLogic.cs

```diff
@@ -25,6 +27,31 @@ public MapEditorLogic(Widget widget, World world, WorldRenderer worldRenderer)
 		{
 			var editorViewport = widget.Get<EditorViewportControllerWidget>("MAP_EDITOR");
 
+			var saveButton = widget.GetOrNull<ButtonWidget>("SAVE_BUTTON");
+			if (saveButton != null)
+			{
+				saveButton.OnClick = () =>
+				{
+					try
+					{
+						var map = worldRenderer.World.Map;
+						map.Save((IReadWritePackage)map.Package);
+						TextNotificationsManager.Debug("Saved current map");
```

Comment:
There is no notifications tray in the map editor so this message doesn't appear anywhere. Could you change this to a transient and add a transient notifications tray? Just add this line to `LoadMapEditorLogic.cs`:
```csharp
Game.LoadWidget(world, "TRANSIENTS_PANEL", editorRoot, new WidgetArgs());
```

And add a full stop to the message to be consistent with the other transients.